### PR TITLE
LTD-2529: Add Exporter notification when licence is revoked

### PR DESCRIPTION
**Pull request by saruniitr**

### CODE REVIEW

File: api/cases/notify.py [hunk 1]
```diff
@@ -51,6 +51,28 @@ def notify_exporter_licence_refused(case):
     )
 
 
+def _notify_exporter_licence_revoked(email, data):
```
Comment (depsiatwal) [inline api/cases/notify.py:54]: any rational of why a private method is required ?

**Review by depsiatwal — APPROVED** [review]
just small comment and need tests fixing other then that LGTM

File: api/cases/notify.py (hunk 1 above)
Comment (saruniitr) [reply api/cases/notify.py:54]: I've just followed the convention from previous examples.

Comment (depsiatwal) [reply api/cases/notify.py:54]: yeah seen that in other PR still wonder why the _ methods :-)

**Pull request merged**